ee/tabby-db/src/users.rs

```diff
@@ -20,13 +20,14 @@ pub struct UserDAO {
     /// To authenticate IDE extensions / plugins to access code completion / chat api endpoints.
     pub auth_token: String,
     pub active: bool,
+    pub avatar: Option<String>,
```

Comment:
Blob itself might not be valid utf-8 string, storing it as `String` will likely generate panic in runtime.

Reply:
Seems some of my changes got lost somewhere, I have just removed the field since it is never used as part of the user struct